webviz_subsurface/containers/_summary_stats.py

```diff
@@ -7,41 +7,69 @@
 from webviz_config.webviz_store import webvizstore
 from webviz_config.common_cache import cache
 from ..datainput import scratch_ensemble
+from plotly.colors import DEFAULT_PLOTLY_COLORS
+import itertools
 
 
 class SummaryStats:
-    '''### Summary statistics
 
-This container visualizes simulation profiles, both per realization and
-statistical plots (min, max, mean, p10, p90).
+    """Summary statistics
 
-* `ensemble`: Which ensemble in `container_settings` to visualize.
-* `sampling`: Optional. Either `monthly` or `yearly`. Default is `monthly`.
-* `title`: Optional title for the container.
-'''
+    Args:
+        ensembles: list of ensemble paths. Length >= 1
+        column_keys: list of pre defined vectors to visualize. Default is none
+        sampling: Optional. Either 'monthly' or 'yearly'. Default is 'monthly'.
+        title: Optional title for the container.
+        history_uncertainty: boolean value if history vector is subjected to
+            uncertainty.
+    Return:
+        dcc.Graph() of either statistics plot or realization plot.
+    Annotations:
+        Function argument-litst are passed as tuple as they are hashed in
+        webviz-store --portable option (list-type is not hashable).
+        Pandas within fmu.ensemlbe expects list. Therefore argument-tuples are
+        converted back into lists within get_smry_data and get_smry_stats
+        functions.
+    """
 
     def __init__(
             self,
             app,
             container_settings,
-            ensemble,
+            ensembles,
+            column_keys=None,
             sampling: str = 'monthly',
-            title: str = 'Simulation time series'):
+            title: str = 'Simulation time series',
+            history_uncertainty: bool = False):
 
         self.title = title
         self.dropwdown_vector_id = 'dropdown-vector-{}'.format(uuid4())
```

Comment:
- Rename to `self.dropdown_vector_id` ?
- Enough to create one uid `self.uid = f'{uuid4()}'`
- Change all string formatting to  f-strings e.g. `f'dropdown-vector-{self.uid}'`